contracts/0.4.24/test_helpers/TestLido.sol

```diff
@@ -50,4 +50,20 @@ contract TestLido is Lido {
     function toLittleEndian64(uint256 _value) public pure returns (uint256 result) {
         return _toLittleEndian64(_value);
     }
+
```

Comment:
Let's rename all contracts with `Test...` names to `Mock...` since they're mocks, not tests